main.go

```diff
@@ -108,26 +212,60 @@ func main() {
 	}
 	filePath := os.Args[1]
 	debug("## Loading " + filePath)
-
-	fileData, err := ioutil.ReadFile(filePath)
+	
+	var subsetPartition map[int][]int
+	err, test := readTestFile(filePath)
 	if err != nil {
 		fatal(err)
 	}
 
-	var test Test
-	var summary Summary
+	rand.Seed(time.Now().UTC().UnixNano())
```

Comment:
Looks like this line is always ran, maybe only want to do so when `RANDOM` partition is requested?